src/envVariables.js

```diff
@@ -137,7 +137,7 @@ let enviromentVariables = {
 		optional: process.env.ENABLE_EMAIL_FOR_REPORT_ISSUE === 'true' ? false : true,
 		default: 'user_issue_reported',
 	},
-	BIG_BLUE_BUTTON_BASE_URL: {
+	BIB_BLUE_BUTTON_BASE_URL: {
```

Comment:
@suman Can you update the code BIG_BLUE_BUTTON_BASE_URL, so it will be proper

Reply:
then no need of merging this pr , its already having the same @rakeshSgr

Reply:
but in code we still refer this BIB_BLUE_BUTTON_BASE_URL

Reply:
update the code where ever we use this key

Reply:
@rakeshSgr ok